src/geometry/manifold/instances/units_of_normed_algebra.lean

```diff
@@ -1,53 +1,64 @@
 /-
 Copyright © 2021 Nicolò Cavalleri. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Nicolò Cavalleri, Heather Macbeth
+Authors: Nicolò Cavalleri, Heather Macbeth, Winston Yin
 -/
 
 import geometry.manifold.smooth_manifold_with_corners
+import geometry.manifold.algebra.lie_group
```

Comment:
Probably the `smooth_manifold_with_corners` import is superseded by the `lie_group` one.